models/actions/run.go

```diff
@@ -22,15 +22,14 @@ import (
 	"code.gitea.io/gitea/modules/util"
 	webhook_module "code.gitea.io/gitea/modules/webhook"
 
-	"github.com/nektos/act/pkg/jobparser"
 	"xorm.io/builder"
 )
 
 // ActionRun represents a run of a workflow file
 type ActionRun struct {
 	ID                int64
 	Title             string
-	RepoID            int64                  `xorm:"index unique(repo_index)"`
+	RepoID            int64                  `xorm:"index unique(repo_index) index(repo_concurrency)"`
```

Comment:
so that the first `index` could be removed?

Reply:
Still the question, the `index` should be removed? It duplicates with `unique`?